react/week5/Github_Users_Search/src/UserRepo.js

```diff
@@ -0,0 +1,28 @@
+import React, {useEffect} from 'react';
+import SearchContext from "./context";
+ import './index.css';
+ import { Link } from 'react-router-dom';
+
+
+ function UserRepo() {
+     const {reposArray, loginName, fetchUserRepos } = React.useContext(SearchContext);
+
+     useEffect(() => {
+         fetchUserRepos(loginName)
+     }, [fetchUserRepos, loginName]);
```

Comment:
I _think_ we can leave the array empty when we only want to fetch data once, from the documentation:

> If you pass an empty array ([]), the props and state inside the effect will always have their initial values. While passing [] as the second argument is closer to the familiar componentDidMount and componentWillUnmount mental model, there are usually better solutions to avoid re-running effects too often. Also, don’t forget that React defers running useEffect until after the browser has painted, so doing extra work is less of a problem.
https://reactjs.org/docs/hooks-effect.html

Reply:
Implemented the changes. Thank you Alicia